### PR TITLE
issue #7158

### PR DESCRIPTION
**Pull request by rachana-webkul**
## Issue Reference
#7158

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
The space should be according to the card content on the home page.

File: packages/Webkul/Product/src/Type/Bundle.php [hunk 1]
```diff
@@ -232,22 +232,28 @@ public function getPriceHtml()
...
 
         if ($prices['from']['regular_price']['price'] != $prices['to']['regular_price']['price']
             || $prices['from']['final_price']['price'] != $prices['to']['final_price']['price']
         ) {
-            $priceHtml .= '<span class="bundle-to">To</span>';
+            $priceHtml .= '<div class="bundle-to">To';
 
             if ($prices['to']['regular_price']['price'] != $prices['to']['final_price']['price']) {
-                $priceHtml .= '<span class="bundle-regular-price">' . $prices['to']['regular_price']['formatted_price'] . '</span>'
+                $priceHtml .= '<span class="bundle-regular-price" style="margin-left: 6px;">' . $prices['to']['regular_price']['formatted_price'] . '</span>'
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Type/Bundle.php:254]: Use classes instead of inline style.

File: packages/Webkul/Product/src/Type/Bundle.php [hunk 2]
```diff
@@ -232,22 +232,28 @@ public function getPriceHtml()
         }
 
         $priceHtml .= '<div class="price-from">';
+         
+        $priceHtml .= '<div class="bundle-to">From';
 
         if ($prices['from']['regular_price']['price'] != $prices['from']['final_price']['price']) {
-            $priceHtml .= '<span class="bundle-regular-price">' . $prices['from']['regular_price']['formatted_price'] . '</span>'
+
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Type/Bundle.php:239]: Remove one line from here.

File: packages/Webkul/Velocity/src/Resources/views/shop/products/price.blade.php [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {!! view_render_event('bagisto.shop.products.price.before', ['product' => $product]) !!}
 
-<div class="product-price">
+<div class="product-price" style="height: 2.5rem;">
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/products/price.blade.php:3]: Use classes.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by rachana-webkul** [comment]
@devansh-webkul 
classes added and spaces resolved, please verify

**Review by devansh-webkul — COMMENTED** [review]
@Sanjaybhattwebkul,

Retest this again as classes added in styling.

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
Please resolve the **conflicts**.

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
**Not fixed** for **mobile** view, Please find the **screenshot** [Here](https://webkul.chatwhizz.com/share/screenshot/63f32576bf991905516bade8)

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
Resolve the **conflicts**.

**Comment by rachana-webkul** [comment]
@Sanjaybhattwebkul 
conflict resolved,please verify

**Comment by rachana-webkul** [comment]
@Sanjaybhattwebkul 
ui changes done, please verify

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
Getting **duplicate commits.** please resolve

**Comment by rachana-webkul** [comment]
@Sanjaybhattwebkul 
Ui fixed ,please verify

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
The **_add to car_**t button is not responsive for mobile.

**Comment by rachana-webkul** [comment]
@Sanjaybhattwebkul 
Button is now responsive according to the mobile view, please verify

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
The product price is overriding the add to cart button in mobile view, please find the screenshot [Here](https://webkul.chatwhizz.com/share/screenshot/63f714a7aac9534fac26b8b5)

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
Now, this is **working fine**, please find the **screenshot** [Here](https://webkul.chatwhizz.com/share/screenshot/63f8452caac9534fac26eaf0)

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by devansh-webkul — APPROVED** [review]

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss [hunk 4]
```diff
@@ -274,7 +274,7 @@
     .product-rating {
         @extend .text-nowrap;
         color: #212529;
-
+        margin-top: -20px;
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss:277]: Add new line after this statement

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 5]
```diff
@@ -47,6 +47,22 @@
...
+            .card-body {
+                    .add-to-cart-btn {
+                        .btn-add-to-cart {
+                            margin-top: -48px;
+                        }
+                    }
+                } 
+            }
+
+        .product-image-container img {
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:63]: Fixed indentation

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 6]
```diff
@@ -951,15 +968,28 @@
...
+                            width: max-content;
+                            padding: 3px 10px !important;
+                            margin-left: 15px !important;
+                        }
+                    }
+                 }
 
-        .product-image-container img {
-            min-height: unset;
+            .product-image-container img {
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:990]: Fixed indentation

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 7]
```diff
@@ -991,7 +1021,19 @@
...
-        max-width: 9rem;
+        &.grid-card {
+            .card-body {
+                margin-top: -17px;
+                    .add-to-cart-btn {
+                        padding: 0;
+                        display: table;
+                            .btn-add-to-cart {
+                                 padding: 3px 7px !important;
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:1032]: Remove line from here

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 8]
```diff
@@ -991,7 +1021,19 @@
...
+                            }
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:1033]: Fixed indentation

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 9]
```diff
@@ -1002,11 +1044,42 @@
...
+                        .btn-add-to-cart {
+                            margin-left: -11px !important;
+                            max-width: 8.75rem !important;
+                            margin-top: 7px;
+                        }
+                    }
+                } 
+            }
 
         .product-image-container img {
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:1060]: Fixed indentation

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 10]
```diff
@@ -1002,11 +1044,42 @@
...
+        .bundle-to {
+            display: flex;
+            flex-wrap: wrap;
+            width: 100px;
+        }
+    }
+}
+
+@media only screen and (max-width: 357px) {
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:1075]: Remove new line

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 11]
```diff
@@ -1051,3 +1124,13 @@
         }
     }
 }
+
+@media only screen and (max-width: 280px) {
+    .product-card-new {
+        max-width: 9rem;
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:1131]: Remove new line

**Comment by Sanjaybhattwebkul** [comment]
@rachana-webkul 
Resolve the conflicts

**Comment by rachana-webkul** [comment]
@Sanjaybhattwebkul @jitendra-webkul 
conflicts and comments resolved,please verify

**Comment by Amitk-webkul** [comment]
@rachana-webkul 

Please resolved the conflicts.

**Review by devansh-webkul — APPROVED** [review]

**Pull request closed without merging**